src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -110,8 +110,8 @@ public void postTenant(TenantAttributes ta, Map<String, String> headers,
         tl.add(p);
       }
       tl.withKey(SAMPLE_KEY).withLead(SAMPLE_LEAD);
+      tl.withPostOnly().withAcceptStatus(400);
```

Comment:
What does `withAcceptStatus(400)` mean? Does that allow creation of records to fail with a 400 status code and the load still be considered a success?

Reply:
@adamdickmeiss please could you help me with my question